scripts/jest.js

```diff
@@ -1,8 +1,9 @@
 const jest = require('jest');
-const config = require('./ui_framework_test_config');
+const config = require('../test/jest/config');
```

Comment:
It feels strange to me to have our config in one directory, the script file in another directory, and the grunt task in another. I'm not clear on the pattern we're trying to follow. Unless there's a consistent pattern applied cross-directory per concern, I think this makes the folder structure more confusing, not less. In which case, I'd prefer to keep the files grouped in a single directory per concern.